packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -110,31 +150,4 @@ const AdvertiserPage = () => {
     );
 };
 
-AdvertiserPage.propTypes = {
-    active_index: PropTypes.number,
-    advert: PropTypes.object,
-    advertiser_info: PropTypes.object,
-    adverts: PropTypes.array,
-    api_error_message: PropTypes.string,
-    counterparty_type: PropTypes.string,
-    error_message: PropTypes.string,
-    form_error_message: PropTypes.string,
-    handleTabItemClick: PropTypes.func,
-    height_values: PropTypes.array,
-    is_loading: PropTypes.bool,
-    is_submit_disabled: PropTypes.bool,
-    item_height: PropTypes.number,
-    modal_title: PropTypes.string,
-    onCancelClick: PropTypes.func,
-    onConfirmClick: PropTypes.func,
-    onMount: PropTypes.func,
-    onTabChange: PropTypes.func,
-    setFormErrorMessage: PropTypes.func,
-    setIsSubmitDisabled: PropTypes.func,
-    setSubmitForm: PropTypes.func,
-    show_ad_popup: PropTypes.bool,
-    showAdPopup: PropTypes.func,
-    submitForm: PropTypes.func,
-};
```

Comment:
Any reason for removing the propTypes?

Reply:
If you check above in the same file, you'll see there are no props for this component. I removed them a long time ago when I added stores but apparently didn't remove the propTypes.